LambChops.md

```diff
@@ -0,0 +1,33 @@
+# Lamb Chops
+
+| *ingredient* | *amount* |
+| --- | --- |
+| Lamb Loin | 8 Lamb (1-inch) |
+| Thyme | 1 Tbsp |
+| Kosher Salt | 3/4 Tsp |
+| Ground Pepper | 1/4 Tsp |
+| Unsalted Butter | 4 Tbsp |
+| Garlic Clove | 1 Large Smashed |
+| Dry White Wine / Low-Sodium Chicken Broth | 1/2 Cup |
+| Lemon Juice | 1 Tsp |
+| Lemon Zest | 1 Tbsp |
+
+Remove Lamb Chops from refrigerator and let it sit for 10 min. Using 1/2 Tsp of salt and the pepper, message into the meat. 
+
+## Prepare Pan and Cook
+
+Melt 2 Tbsp of butter into a 12-in skillet over medium-high heat. 
+
+Add the Lamb Chops and cook until a brown crust forms at the bottom. 4-6 minutes depending on the thickness.
+
+Flip Lamb Chops and continue to cook until tickest part of the middle is 145&deg;F, about 4-6 more minutes.
```

Comment:
nit: same caps
Nice code there for the temp!

Reply:
Done.